src/riscv.rs

```diff
@@ -101,60 +101,31 @@ MTVAL=0x{:08x}
     }
 }
 
-/// Get an array of backtrace addresses.
+/// Get an iterator of backtrace addresses.
 ///
 /// This needs `force-frame-pointers` enabled.
-pub fn backtrace() -> [Option<usize>; MAX_BACKTRACE_ADRESSES] {
+#[inline(never)]
```

Comment:
Optimizer doing its own thing could make things unpredictable, so by disallowing inlining I hope we always skip the correct number of frames